apps/fulfillment-e2e/src/integration/picking-lists-search.cy.ts

```diff
@@ -0,0 +1,59 @@
+import { PickingListsFragment } from '../support/page_fragments/picking-lists.fragment';
+
+const pickingListsFragment = new PickingListsFragment();
+
+describe('Picking Lists Search', () => {
+  beforeEach(() => {
+    cy.clearIndexedDB();
+    cy.login();
+
+    pickingListsFragment.getSearch().click();
+  });
+
+  it('should check success search flow', () => {
+    pickingListsFragment.getWrapper().should('be.visible');
+    pickingListsFragment.getNoItemsFallback().should('be.visible');
+    pickingListsFragment
+      .getNoItemsFallbackHeading()
+      .should('contain.text', 'Search by order ID');
```

Comment:
In the future, we might have to update this text, and if we keep it in code and in the e2e test -> we will have to update both places

We can replace it with a visibility check to make it more stable